src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,70 @@
+import {useState} from 'react';
+import Parse from 'parse';
+import { useSelector } from 'react-redux';
+import { ApplicationState } from 'types/store';
+import MenuButton from 'components/Menu/MenuItem/MenuButton';
+import MenuToggle from 'components/Menu/MenuItem/MenuToggle';
+import { ReactComponent as AddImageIcon } from 'assets/icon-addimage.svg';
+import { ReactComponent as AddStickerIcon } from 'assets/icon-addsticker.svg';
+import { ReactComponent as CheckIcon } from 'assets/icon-check.svg';
+import { ReactComponent as SettingsIcon } from 'assets/icon-settings.svg';
+import { ReactComponent as ColumnIcon } from 'assets/icon-column.svg';
+import { ReactComponent as FocusIcon } from 'assets/icon-focus.svg';
+import { ReactComponent as TimerIcon } from 'assets/icon-timer.svg';
+import { ReactComponent as VoteIcon } from 'assets/icon-vote.svg';
+import { ReactComponent as ToggleSettingsMenuIcon } from 'assets/icon-toggle-settings-menu.svg';
+import { ReactComponent as ToggleAddMenuIcon } from 'assets/icon-toggle-add-menu.svg';
+import './Menu.scss';
+
+function Menu() {
```

Comment:
I would rather name this component `MenuBars` or something like that. Just naming it `Menu` is maybe to generic